apps/server/src/modules/board/controller/element.controller.ts

```diff
@@ -141,4 +143,18 @@ export class ElementController {
 
 		return response;
 	}
+
+	@ApiOperation({ summary: 'Check permission for a drawing element.' })
+	@ApiResponse({ status: 201 })
+	@ApiResponse({ status: 400, type: ApiValidationError })
+	@ApiResponse({ status: 403, type: ForbiddenException })
+	@ApiResponse({ status: 404, type: NotFoundException })
+	@HttpCode(201)
+	@Get(':drawingName/permission')
+	async checkDrawingPermission(
```

Comment:
I don't see anything that is special for a drawing in the authorization logic. You just check it like any other element. So I don't think "drawing" should be in any of the names.